lua/lazyvim/plugins/extras/lang/typst.lua

```diff
@@ -0,0 +1,62 @@
+return {
+  recommended = function()
+    return LazyVim.extras.wants({
+      ft = { "typst" },
+    })
+  end,
+
+  {
+    "nvim-treesitter/nvim-treesitter",
+    opts = {
+      ensure_installed = { "typst" },
+    },
+  },
+
+  {
+    "neovim/nvim-lspconfig",
+    opts = {
+      servers = {
+        tinymist = {
+          single_file_support = true, -- Fixes LSP attachment in non-Git directories
+          root_dir = LazyVim.root.get,
```

Comment:
don't set this.
Also no need to enable single_file_support

Reply:
If `single_file_support` is not enabled, the LSP won't attach to the buffer when it’s not in a Git repository. See the attached screenshot for reference.

![image](https://github.com/user-attachments/assets/5df8fd63-170b-4800-9dc6-6392b59d5664)

Reply:
ok, got it. enable then indeed